bot.go

```diff
@@ -280,6 +279,9 @@ func (b *Bot) handler(c *irc.Client, m *irc.Message) {
 	}
 
 	b.mux.HandleEvent(b, r)
+	timer.Done()
+
+	r.LogTimings(b.log)
```

Comment:
What's the proper way here to say "if using debug logging or higher then call this method"?